blazor/rich-text-editor/import-export.md

```diff
@@ -125,18 +125,18 @@ The Rich Text Editor allows you to load an external text file in to the editor.
 {% endhighlight %}
 {% endtabs %}
 
-![Import to text file](./images/blazor-import-text.png)
+![Blazor RichTextEditor import to text file](./images/blazor-richtexteditor-import-text.png)
 
 > [View Sample in GitHub](https://github.com/SyncfusionExamples/import-text-file-to-blazor-rich-text-editor).
 
 ## Export to RTF file
 
-You can able to export the RTE content to RTF format by using the [Syncfusion.DocIO](https://libraries.io/nuget/Syncfusion.DocIO.NET) libraries. 
+Using the [Syncfusion.DocIO](https://libraries.io/nuget/Syncfusion.DocIO.NET) libraries, you can export RTE content to RTF format.
 
-Use the following code to export the RTF file. While cliking on the export button you can call make the RTE content to the RTF file by using **Syncfusion.DocIO** libraries.
+Use the following code to export the RTF file: When you click the export button, you can use the **Syncfusion.DocIO** libraries to convert the RTE content to an RTF file.
```

Comment:
Click **Export**, and use **Syncfusion.DocIO** libraries to convert the RTE content to an RTF file. Use the following code to export the RTF file.